response/page_path_test.go

```diff
@@ -0,0 +1,265 @@
+package response
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type TestCase struct {
+	uri      string
+	expected string
+}
+
+func TestGetPagePath(t *testing.T) {
+	Convey("Given a list of URIs", t, func() {
+		ctx := context.Background()
+
+		visualisationsEndpoints := []TestCase{
+			{
```

Comment:
its very unlikely but i could imagine you get a path like "http://www.ons.gov.uk/eonomy/..." in that case it should remove the protocol and host and port

Reply:
This cannot happen, we get the path from the request struct, so the host and protocol are never part of it.